target/linux/ramips/dts/mt7621_iodata_wn-xx-xr.dtsi

```diff
@@ -120,10 +120,36 @@
 	};
 };
 
-&ethernet {
+&gmac0 {
 	mtd-mac-address = <&factory 0xe000>;
 };
 
+&switch0 {
+	ports {
+		port@0 {
+			status = "okay";
+			label = "wan";
+			mtd-mac-address = <&factory 0xe006>;
```

Comment:
This is the label mac address (see 02_network).
I suspect we can add a label to this port and then use it with label-mac-device?
Otherwise, we need to keep the label_mac in 02_network for
-	iodata,wn-ax1167gr2
-	iodata,wn-dx1167r

Reply:
Just do the same as for jcg_jhr-ac876m here ... :-)